tareas/clase-5/index3.js

```diff
@@ -0,0 +1,31 @@
+let formulario = document.getElementsByClassName("tiempovideos");
+let agregarClase = document.getElementById("agregarform");
+let cantidadClases = 1;
+let clonDeFormulario = formulario[0].cloneNode(true); //tengo que clonar el clon del formulario sino me clona tambien los valores que estan dentro del campo de texto
+
+
+agregarClase.onclick = () => {
+    cantidadClases ++;
+    document.querySelector("div").appendChild(clonDeFormulario.cloneNode(true));
+    document.getElementsByClassName("nombreclase")[document.getElementsByClassName("nombreclase").length-1].innerHTML = "Clase " + cantidadClases; // en el lado izquierdo de la asignacion selecciono el ultimo elemento del array que contiene la clase "nombreclase"
```

Comment:
Podés usar el pseudo-selector `last-of-type` para traer el último elemento en vez de traer todos y sólo elegir el último.

```suggestion
    document.querySelector("nombreclase:last-of-type").innerHTML = "Clase " + cantidadClases;
```

Reply:
De hecho, esto no funciona. Funciona sólo si usás el nombre del tag, mala mía. https://css-tricks.com/almanac/selectors/l/last-of-type/

Reply:
¡Gracias por el aviso! ahí lo cambio.